build.sbt

```diff
@@ -41,7 +41,7 @@ lazy val rootSettings =
     dynver ~= (v => s"${sparkVersionSystem}_$v")
   )
 
-lazy val zioVersion = "1.0.16"
+lazy val zioVersion = "2.0.0"
```

Comment:
should we be supporting zio 1.x and zio 2.x ? Like for spark versions ?

Reply:
Hmm, we could do this from separate branches (like ZIO does) or separate code somehow but that's much more complex refactoring.

Reply:
hmm ok

Reply:
Unless, You know other way to make that work ofc :)